griptapecli/commands/skatepark.py

```diff
@@ -65,9 +65,7 @@ def start(
     port: int,
 ) -> None:
     """Starts the Griptape server."""
-    uvicorn.run(
-        "griptapecli.core.skatepark:app", host=host, port=port, reload=False, workers=4
-    )
+    uvicorn.run("griptapecli.core.skatepark:app", host=host, port=port, reload=False)
```

Comment:
what does `workers` default to?

Reply:
`1` but I will explicitly set it in an upcoming PR.